code/ui/generalhud/playerinfo/PlayerInfoDisplay.cs

```diff
@@ -0,0 +1,56 @@
+using Sandbox;
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+using TTTReborn.Player;
+using TTTReborn.Roles;
+using TTTReborn.Settings;
+
+namespace TTTReborn.UI
+{
+    public class PlayerInfoDisplay : Panel
+    {
+        private Panel _healthPanel;
+        private Label _healthLabel;
+        private Panel _creditPanel;
+        private Label _creditLabel;
+        private Panel _staminaPanel;
+
+        public PlayerInfoDisplay() : base()
+        {
+            StyleSheet.Load("/ui/generalhud/playerinfo/PlayerInfoDisplay.scss");
+
+            AddClass("background-color-primary");
+            AddClass("rounded");
+            AddClass("opacity-heavy");
+            AddClass("text-shadow");
+
+            _healthPanel = new Panel(this);
+            _creditPanel = new Panel(this);
+
+            _healthLabel = _healthPanel.Add.Label();
+            _healthLabel.AddClass("info");
+
+            _creditLabel = _creditPanel.Add.Label();
+            _creditLabel.AddClass("info");
+
+        }
+
+        public override void Tick()
+        {
+            base.Tick();
+
+            if (Local.Pawn is not TTTPlayer player)
+            {
+                return;
+            }
+
+            Enabled = Local.Pawn is TTTPlayer && !player.IsSpectator && !player.IsSpectatingPlayer;
```

Comment:
So you removed first person spectator support?

Reply:
This was a mistake.